.github/workflows/publish-helm-chart.yml

```
@@ -75,6 +75,61 @@ jobs:
             run: |
               helm lint ${{matrix.chartpath}}
 
+          - name: Helm Chart Version Check
+            run: |
+              # old_chart_version diffs commit that triggered workflow against previous 
+              # commit. Pulls out version number of the existing chart or "old chart".
+              # The git diff gives spits out both version numbers, head -2 | tail -1
+              # Grabs the value on the second line. 
+              old_chart_version=$(git diff ${{ github.sha }} \
+               ${{github.event.before}} -- ${{matrix.chartpath}} | 
+               grep 'version:' | awk '/version:/ {print $2}' | head -2 | 
```

```suggestion
               awk '/version:/ {print $2}' | head -2 | 
```

The grep is superfluous here (and below)

Could you also split up each fragment of the pipeline onto its own separate line? See https://github.com/osg-htc/helm-charts/pull/22/files#diff-dce247c65e791dfbb2975e023ffc0d20d556f48be798bc6c462ba808ccbbc26fR40-R48 as an example. It's nice to split up options for a single command and line them up and line up the commands on separate lines for readability